backend/src/schema/resolvers/users.js

```diff
@@ -174,6 +174,7 @@ export default {
     },
     DeleteUser: async (object, params, context, resolveInfo) => {
       const { resource } = params
+      const { id } = params
```

Comment:
This `id` should be set correct in the GQL query from the frontend. And not in the `permissionsMiddleware`
Is this not working for what ever reason?

In case of the current user deletes its own account:

```
id = currentuser.id
```

In case of an admin the user to delete:

```
id = user.id
```